model_zoo/models/densenet/generator_densenet.py

```diff
@@ -1,7 +1,13 @@
 # Generate DenseNet prototext
-# See https://github.com/pytorch/vision/blob/6f7e26be1018344a4d1015370dcedba7b772bbc1/torchvision/models/densenet.py
+# See https://github.com/pytorch/vision/blob/master/torchvision/models/densenet.py
 # See "Densely Connected Convolutional Networks" by Huang et. al
 
+# Note that `--learn_rate` for the sgd optimizer should be set to 0.1 for this model.
+
+import sys
+sys.path.insert(0, '..')
+import model_generator
+
 ### DenseNet Specific #########################################################
```

Comment:
Can remove this line now that general code is in `model_generator.py` (also, rename to `generator_model.py` to match naming structure).